qa/suites/fs/upgrade/featureful_client/upgraded_client/tasks/4-compat_client.yaml

```diff
@@ -13,3 +13,11 @@ tasks:
     clients:
       client.0: False
       client.1: True
+# cleanup evicted client so there's no trouble later.
```

Comment:
nit: add a comment that _only_ client.0 is upgraded and client.1 is evicted by the mds due to missing feature compat set.

Reply:
Done - https://github.com/ceph/ceph/pull/46988/files#diff-12f10acf04fc223775779597acad35c68ffd3e591bc4eb79df53a4b224dceabcR12.